x-pack/plugins/security_solution/server/endpoint/routes/actions/response_actions.test.ts

```diff
@@ -694,6 +695,13 @@ describe('Response actions', () => {
             {
               id: `case-${counter++}`,
               title: 'case',
+              createdAt: '2022-10-31T11:49:48.806Z',
```

Comment:
The mock implementation type was complaining so I updated it to match the new type.